### PR TITLE
[TwigBundle] Removed capability to load Twig template from legacy directories

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Ref https://github.com/symfony/symfony/pull/28891

### CODE REVIEW

**Comment by yceruto** [comment]
(AppVeyor failure unrelated)

File: src/Symfony/Bridge/Twig/Command/DebugCommand.php [hunk 1]
```diff
@@ -38,19 +38,17 @@ class DebugCommand extends Command
     private $projectDir;
     private $bundlesMetadata;
     private $twigDefaultPath;
-    private $rootDir;
     private $filesystemLoaders;
     private $fileLinkFormatter;
 
-    public function __construct(Environment $twig, string $projectDir = null, array $bundlesMetadata = [], string $twigDefaultPath = null, string $rootDir = null, FileLinkFormatter $fileLinkFormatter = null)
+    public function __construct(Environment $twig, string $projectDir = null, array $bundlesMetadata = [], string $twigDefaultPath = null, FileLinkFormatter $fileLinkFormatter = null)
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Twig/Command/DebugCommand.php]: we need an upgrade path before changing the order or arguments

**Review by nicolas-grekas — CHANGES_REQUESTED** [review]

File: src/Symfony/Bridge/Twig/Command/DebugCommand.php (hunk 1 above)
Comment (yceruto) [reply src/Symfony/Bridge/Twig/Command/DebugCommand.php]: Opps, sure I missed that

Comment (yceruto) [reply src/Symfony/Bridge/Twig/Command/DebugCommand.php]: Done in https://github.com/symfony/symfony/pull/31688

**Comment by yceruto** [comment]
Waiting for https://github.com/symfony/symfony/pull/31688

Status: Needs Work

**Review by xabbuh — APPROVED** [review]

**Comment by yceruto** [comment]
Rebased and ready!

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**